.github/workflows/actions.yml

```diff
@@ -66,3 +66,33 @@ jobs:
       - name: Verify Conda Package
         run: |
           conda verify noarch/imars3d*.tar.bz2
+
+  trigger-deploy:
+    runs-on: ubuntu-22.04
+    needs: [linux, build-conda]
+    # only trigger deploys from protected branches
+    if: ${{ github.ref_protected || github.ref_type == 'tag' }}
+    steps:
+      - name: Determine Environment
+        uses: neutrons/branch-mapper@main
+        id: conda_env_name
+        with:
+          prefix: imars3d
+
+      - name: Trigger deploy
+        id: trigger
+        uses: eic/trigger-gitlab-ci@v2
+        with:
+          url: https://code.ornl.gov
+          token: ${{ secrets.GITLAB_TRIGGER_TOKEN }}
+          project_id: 11848
+          ref_name: main
+          variables: |
+            PLAY="update"
+            CONDA_ENV="${{ steps.conda_env_name.outputs.name }}"
```

Comment:
One quick question, is this section for both `qa` and `stable`?
Also, how do we trigger the deployment of the `next` branch from Github?

Reply:
```
    if: ${{ github.ref_protected || github.ref_type == 'tag' }}
```
for the job means "protected branches or tags." The "Determine Environment" section converts the refspec to a conda environment name.